x/qgb/keeper/keeper_data_commitment.go

```diff
@@ -21,19 +21,19 @@ func (k Keeper) NextDataCommitment(ctx sdk.Context) (types.DataCommitment, error
 	if err != nil {
 		return types.DataCommitment{}, err
 	}
-	// for a data commitment window of 400, the ranges will be: 1-400;401-800;801-1200
+	// for a data commitment window of 400, the ranges will be: [1-401[, [401-801[, [801-1201[
```

Comment:
```suggestion
	// for a data commitment window of 400, the ranges will be: [1-401], [401-801], [801-1201]
```

Reply:
I think because the range is exclusive, we refer to it by `[`, no? the mathematical notation for an open ended interval?

Reply:
<img width="734" alt="Screenshot 2023-05-22 at 11 40 31 AM" src="https://github.com/celestiaorg/celestia-app/assets/3699047/cff647a8-10c9-4a13-93b5-b441a4891ccb">

https://courses.lumenlearning.com/suny-osalgebratrig/chapter/domain-and-range/#:~:text=Parentheses%2C%20(%20or%20)%2C%20are,value%20is%20included%2C%20called%20inclusive.

So if you want to indicate that the range is end-exclusive, you could use `)`. For example: 

```go
	// for a data commitment window of 400, the ranges will be: [1-401), [401-801), [801-1201)
```

Reply:
Didn't know :D french vs english notation haha will make the change